src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -333,6 +335,17 @@ export default {
                 return found;
             });
 
+            this.timestampAndUpdate();
+        },
+        async timestampAndUpdate() {
+            const user = await this.openmct.user.getCurrentUser();
+
+            if (user !== undefined) {
+                this.entry.modifiedBy = 'Unknown';
```

Comment:
Is this a magic string we'll need to use elsewhere?

Reply:
It's a magic value, yup. I'll make it a const :) Not sure if we will ever need it anywhere else.